internal/handler/wallet/sign.go

```diff
@@ -53,7 +53,8 @@ func (s *walletSign) Handle(params operations.SignParams) middleware.Responder {
 		return resp
 	}
 
-	op, _ := base64.StdEncoding.DecodeString(params.Body.Operation.String())
+	op, err := base64.StdEncoding.DecodeString(params.Body.Operation.String())
+	fmt.Println("base64 decoding error: %w", err)
```

Comment:
no, here you print an error message even when there is no error. What you need to do is to check is the error is nil and if no, handle the error